desktop/window-handlers/external-links/index.js

```diff
@@ -103,7 +124,46 @@ module.exports = function( webContents ) {
 
 		const openUrl = format( parsedUrl );
 
-		log.info( `Using system default handler for URL: ${ openUrl }` );
 		openInBrowser( event, openUrl );
 	} );
+
+	ipc.on( 'cannot-open-editor', async( event, info ) => {
+		const { origin, editorUrl } = info;
+		const parsedURL = new URL( origin );
+
+		const promptWpAdminAuth = () => {
+			try {
```

Comment:
what's the source of the exception we're catching?

Reply:
I think being overly paranoid about showMessageBox failing, but you're right - this is probably unnecessary.